src/Orleans.Core/Configuration/Options/ServicePointOptions.cs

```diff
@@ -0,0 +1,13 @@
+
+namespace Orleans.Hosting
+{
+    public class ServicePointOptions
```

Comment:
if `ServicePointOptions` is only azure storage related , maybe this is not the place it belongs. and also line 637 - 643 in silo.cs should be moved to azure extensions maybe

Reply:
Service point manager seems to be part of .net, not azure.  I think it's only used by azure storage, but it's likely used by other things.  IMO, this should not be part of Orleans at all, but let's look at removing it in a future pass, after we've examined the implications.

Reply:
>  IMO, this should not be part of Orleans at all, but let's look at removing it in a future pass, after we've examined the implications.

👍